repl/format.go

```diff
@@ -37,6 +37,9 @@ func FormatValue(v cty.Value, indent int) string {
 			return fmt.Sprintf("null /* %s */", ty.FriendlyName())
 		}
 	}
+	if v.IsMarked() {
+		return "(sensitive)"
+	}
```

Comment:
This probably doesn't practically matter, but I'd expect a sensitive variable with a `null` value to still display `(sensitive)` in the console, if only because I think it would do so in a plan. I think checking for the mark here instead of before the `if v.IsNull()` branch means it displays `null` instead.

Reply:
Excellent point! And easy enough to move that ordering.